codemod/spotlight_util.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2007-2008 Facebook
+
+from Cocoa import *
+
+
+class SpotlightUtil(object):
+    """
+    Searches a directory hierarchy using OS/X spotlight
+    """
+
+    @staticmethod
```

Comment:
Worked out to be easier to promote SpotlightUtil to a first class object, so we can be threadsafe in the asynchronous query